### PR TITLE
JDK-8301092 - Add benchmark for CRC32

### PR DESCRIPTION
**Pull request by asgibbons**
Adding a performance benchmark test for CRC32.  This does exactly the same test as for CRC32C.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8301092](https://bugs.openjdk.org/browse/JDK-8301092): Add benchmark for CRC32

### Reviewers
 * [Eric Caspole](https://openjdk.org/census#ecaspole) (@ericcaspole - Committer) ⚠️ Review applies to [0f098760](https://git.openjdk.org/jdk/pull/12199/files/0f098760583b42842e3b0be0a68c51841b742a8a)
 * [Sandhya Viswanathan](https://openjdk.org/census#sviswanathan) (@sviswa7 - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/12199/head:pull/12199` \
`$ git checkout pull/12199`

Update a local copy of the PR: \
`$ git checkout pull/12199` \
`$ git pull https://git.openjdk.org/jdk pull/12199/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 12199`

View PR using the GUI difftool: \
`$ git pr show -t 12199`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/12199.diff">https://git.openjdk.org/jdk/pull/12199.diff</a>

</details>

### CODE REVIEW

**Review by ericcaspole — APPROVED** [review]
Looks good to me, thanks!

File: test/micro/org/openjdk/bench/java/util/TestCRC32.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2021, 2022, 2023, Oracle and/or its affiliates. All rights reserved.
```
Comment (sviswa7) [inline test/micro/org/openjdk/bench/java/util/TestCRC32.java:2]: The copyright line should be as follows:
 * Copyright (c) 2021,  2023, Oracle and/or its affiliates. All rights reserved.

Rest of the patch looks good to me.

Comment (asgibbons) [reply test/micro/org/openjdk/bench/java/util/TestCRC32.java]: Fixed.

**Review by sviswa7 — APPROVED** [review]

**Pull request closed without merging**